app/experimenter/nimbus-ui/src/components/AppErrorBoundary/index.tsx

```diff
@@ -26,11 +26,11 @@ class AppErrorBoundary extends React.Component {
 
   render() {
     const { error } = this.state;
-    return error ? <AppErrorAlert {...{ error }} /> : this.props.children;
+    return error ? <AppErrorAlert /> : this.props.children;
   }
 }
 
-export const AppErrorAlert = ({ error }: { error: Error }) => {
```

Comment:
Hmm... looks like this got snipped out because we're not actually displaying anything related to the specific error. Do we want this removed - or do we actually want to do something in the alert with the error?

Reply:
Oh why did github throw out my comment.  Sorry anyways, yes the error isn't used so it gets caught by `no-unused-vars`.  We could display it? I tried not to change the behaviour of any components.